lua/registereditor/internals.lua

```diff
@@ -57,21 +58,19 @@ local function open_editor_window(reg)
     vim.api.nvim_buf_set_lines(0, 0, -1, false, buf_lines)
 
     vim.bo.modified = false
-    
+
     -- Special readonly registers
     if reg:match("[.:%%#]") then
         vim.bo.readonly = true
     end
 
-    vim.api.nvim_create_autocmd({"BufWriteCmd"}, {
+    vim.api.nvim_create_autocmd({ "BufWriteCmd" }, {
         buffer = 0,
         callback = function()
             vim.bo.modified = false
             set_register(reg)
-        end
+        end,
```

Comment:
This `,` bugs me :smile: but I notice that running `stylua internals.lua` is what adds it.

I'm new to stylua, does it have a rule to avoid the trailing commas?

Reply:
Based on [this table](https://github.com/JohnnyMorganz/StyLua?tab=readme-ov-file#options), I don't think trailing comma is an option. I think there are other Lua formatters out there, but stylua is pretty popular. And you don't have to use stylua if you don't want to, but there are a lot of benefits to having a specified autoformat syntax.

Reply:
Ok, it looks like stylua takes a hard stance on this:

https://github.com/JohnnyMorganz/StyLua/issues/526

Let's accept the trailing commas.